### PR TITLE
app header mobile responsive

### PR DESCRIPTION
**Pull request by jjjhill**
image
image
image

### CODE REVIEW

File: src/components/UniverseBalances.vue [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
-      <p><strong>{{totalCryptozSupply}} NFTs </strong> in the Universe</p>
-    </b-col>
-  </b-row>
-  
+  <div class="row-col">
+    <p>
+      <strong>{{totalCzxpSupply}} CZXP
+        {{' '}}<img class="czxp-logo" src="./assets/cryptokeeper_coin_binance.svg" />tokens
+      </strong>
+      {{' '}}in the Universe
```
Comment (ryanprice) [inline src/components/UniverseBalances.vue:7]: What does {{' '}} do ?

**Review by ryanprice — APPROVED** [review]
Tested  in chrome dev tools 
Works nicely.. the whole screen seems to be slightly wider than the viewport, will test on the device once deployed to main

File: src/components/UniverseBalances.vue (hunk 1 above)
Comment (jjjhill) [reply src/components/UniverseBalances.vue:7]: just adds a space character

**Comment by jjjhill** [comment]
> Tested in chrome dev tools
> Works nicely.. the whole screen seems to be slightly wider than the viewport, will test on the device once deployed to main

yeah seemed to be an existing issue, vue bootstrap acting weird or something

File: src/components/UniverseBalances.vue (hunk 1 above)
Comment (ryanprice) [reply src/components/UniverseBalances.vue:7]: o hah.. kinda wondered that  you can also use html &nbsp;

Comment (ryanprice) [reply src/components/UniverseBalances.vue:7]: hah that worked too well use HTML & nbsp;

**Pull request merged**